### PR TITLE
Removing the SimplePush subsystem

### PR DESCRIPTION
**Pull request by matzew**
for resource cleaning up, we are removing the SP server from teh cartridge

### CODE REVIEW

**Comment by matzew** [comment]
@fjuma @sebastienblanc @kpiwko can you test / review ?

**Comment by matzew** [comment]
@danbev FYI in case you feel like it :-)

File: bin/control [hunk 1]
```diff
@@ -377,41 +376,6 @@ function threaddump() {
...
-
-  cleanup_dump
-}
-
-function cleanup_dump {
-  rm -f $OPENSHIFT_DATA_DIR/aerogear_push_token_key
-  rm -f $OPENSHIFT_DATA_DIR/standalone.snapshot.xml
-}
-
-
```
Comment (fjuma) [inline bin/control]: Instead of removing all of these hooks, perhaps only the SimplePush parts should be removed? (i.e., lines 382, lines 396-400, and line 410 can be removed) This way, the standalone.xml file will still get backed up when performing snapshot/restore operations so that a user can still refer to it in case he had made any modifications to it.

**Comment by fjuma** [comment]
The rest of the changes look good to me.

**Comment by matzew** [comment]
@fjuma thanks for the suggestions - applied them


@sebastienblanc @kpiwko any other comments ?

File: metadata/manifest.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Name: aerogear-push
 Cartridge-Short-Name: AEROGEAR_PUSH
 Display-Name: AeroGear UnifiedPush Server 1.0.3 and WildFly
-Description: "Provides the AeroGear UnifiedPush Server, a server that allows sending push notifications to different mobile platforms, and the AeroGear SimplePush Server, an implementation of Mozilla's SimplePush Protocol. Runs on JBoss Application Server 7."
+Description: "Provides the AeroGear UnifiedPush Server, a server that allows sending push notifications to different mobile platforms."
 Version: '1.0.3'
```
Comment (kpiwko) [inline metadata/manifest.yml]: Should not version be bumped here? This is quite significant change.

File: README.md [hunk 3]
```diff
@@ -64,11 +47,6 @@ of use:
 
     OPENSHIFT_AEROGEAR_PUSH_IP                         The IP address used to bind JBossAS
     OPENSHIFT_AEROGEAR_PUSH_HTTP_PORT                  The JBossAS listening port
-    OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY                  The token key for the SimplePush Server
-    OPENSHIFT_AEROGEAR_PUSH_CLUSTER_PORT
```
Comment (kpiwko) [inline README.md]: Where the other env vars made redundant as the part of previous task? They don't seem related to SimplePush to me.

Comment (matzew) [inline README.md]: @kpiwko I *think* they were introduced with the advent of SimplePush.

@fjuma makes sense to keep them or good to remove?

File: metadata/manifest.yml (hunk 2 above)
Comment (matzew) [inline metadata/manifest.yml]: good point :-)

Comment (matzew) [inline metadata/manifest.yml]: Version or Cartridge-Version ? :)

Comment (matzew) [inline metadata/manifest.yml]: I think it's cartridge-version to bump up :)

File: README.md (hunk 3 above)
Comment (fjuma) [inline README.md]: OPENSHIFT_AEROGEAR_PUSH_TOKEN_KEY was introduced for SimplePush.
OPENSHIFT_AEROGEAR_PUSH_CLUSTER_PORT isn't being used anymore so it can be removed (it used to be referenced in the jgroups subsystem configuration). The MESSAGING and REMOTING ports can probably also be removed from the README since they are not specific to the Push cartridge (they're related to WildFly).

File: metadata/manifest.yml (hunk 2 above)
Comment (fjuma) [inline metadata/manifest.yml]: Yes, cartridge-version :)

File: metadata/manifest.yml [hunk 4]
```diff
@@ -104,13 +102,6 @@ Endpoints:
     Private-Port-Name: CLUSTER_PORT
     Private-Port:      7600
     Public-Port-Name:  CLUSTER_PROXY_PORT
-  - Private-IP-Name:   IP
-    Private-Port-Name: MESSAGING_PORT
-    Private-Port:      5445
-    Public-Port-Name:  MESSAGING_PROXY_PORT
-  - Private-IP-Name:   IP
-    Private-Port-Name: MESSAGING_THROUGHPUT_PORT
-    Private-Port:      5455
```
Comment (matzew) [inline metadata/manifest.yml]: @fjuma should we keep the "MESSAGING_" in here ?

Comment (fjuma) [inline metadata/manifest.yml]: @matzew Just checked the WildFly cartridge's manifest file, it no longer includes the "MESSAGING_" variables here, so I think you should be able to remove it as well.

Comment (matzew) [inline metadata/manifest.yml]: @fjuma perfect! Thanks!

@kpiwko good to merge ?

**Comment by kpiwko** [comment]
@matzew looks good to me
